packages/Organisms/Markup/index.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <!-- Keep app-wysiwyg class here for backward compatibility, as some project may use that class for custom style -->
+  <div class="rpl-markup tide-wysiwyg app-wysiwyg">
+    <!-- Use dynamic component to render string template in runtime -->
+    <component :is="{template:getTemplate}"/>
+  </div>
+</template>
+
+<script>
+import { markupTranspiler } from './markup-transpiler'
+
+// To make runtime build work, we need to register the component globally
+import Vue from 'vue'
+import RplButton from '@dpc-sdp/ripple-button'
+import RplDocumentLink from '@dpc-sdp/ripple-document-link'
+import {RplTextLink} from '@dpc-sdp/ripple-link'
+
+Vue.component('rpl-button', RplButton)
+Vue.component('rpl-document-link', RplDocumentLink)
+Vue.component('rpl-text-link', RplTextLink)
+
+export default {
+  name: 'RplMarkup',
+  props: {
+    'html': String,
+    'plugins': { type: Array }
+  },
+  computed: {
+    getTemplate () {
+      // Run our transpile to turn html into vue template.
+      const html = `<div class="rpl-markup__inner">${this.html}</div>`
+      const plugins = this.plugins || this.rplOptions.markupPlugins
+
+      if (plugins && plugins.length > 0) {
+        const template = markupTranspiler(html, plugins)
+        return template
+      }
+      return html
+    }
+  }
+}
+</script>
+
+<style lang="scss">
+@import "~@dpc-sdp/ripple-global/style";
+
+.rpl-markup {
```

Comment:
Most of these styles are also shared with the `fieldRplmarkup.vue` field component - we should remove the styles from `fieldRplmarkup.vue`. There are some differences too - mainly `fieldRplmarkup.vue` adds styles for tide-external-link - this should be moved somewhere.. not sure where.

Reply:
@dylankelly @alan-cole As https://digital-engagement.atlassian.net/browse/SDPA-1253 solution mentioned, if we added the link plugin in wysiwyg then we need to remove the tide external links code. The external links should be handled by Ripple link.